shuttleofx_client/static_tmp/scripts/render.js

```diff
@@ -208,7 +208,8 @@ $(document).ready(function() {
             }),
         })
         .done(function(data){
-			$("#viewer img#originalPic").attr("src", "/resource/" + selectedResource);
+            var selectedResourceName = selectedResource.split(".")[0];
```

Comment:
Add a comment to explain that we ask the png proxy, so we remove the original file extension.